app/views/user_favorites/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render partial: "/partials/navbar" %>
+<h1>UserFavorites#index</h1>
+
+<p>Find me in app/views/user_favorites/index.html.erb</p>
```

Comment:
what is this doing?

Reply:
this is just adding the nav-bar to the `user_favorites` index page. The rest of the test is auto generated by `rails g model`